bundles/org.openhab.binding.remehaheating/src/main/java/org/openhab/binding/remehaheating/internal/RemehaHeatingBindingConstants.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.remehaheating.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link RemehaHeatingBindingConstants} class defines common constants used across the binding.
+ * 
+ * This class contains:
+ * - Thing type UIDs for supported devices
+ * - Channel identifiers for all supported channels
+ * - Configuration parameter names
+ * - DHW mode constants
+ *
+ * @author Michael Fraedrich - Initial contribution
+ */
+@NonNullByDefault
+public class RemehaHeatingBindingConstants {
+
+    private static final String BINDING_ID = "remehaheating";
+
+    // Thing Type UIDs
+    /** Thing type UID for Remeha boiler */
+    public static final ThingTypeUID THING_TYPE_BOILER = new ThingTypeUID(BINDING_ID, "boiler");
+
+    // Channel identifiers
+    /** Current room temperature channel */
+    public static final String CHANNEL_ROOM_TEMPERATURE = "roomTemperature";
```

Comment:
Please adjust to the `lower-case-hyphen` naming convention for channel id,s channel type id's